Readme.txt

```diff
@@ -241,7 +241,11 @@ LOG WINDOW
   CL    - incorrect Arrl Field Day Classification in your log.
   NAME  - incorrect Name in your log.
   SEC   - incorrect ARRL Section in your log.
-  ST    - incorrect State in your log.
+  ST    - incorrect State/Provence in your log.
+  QTH   - incorrect location information.
+  ZN    - incorrect CQ-Zone or ITU-Zone information.
+  SOC   - incorrect Society information (JARL)
```

Comment:
SOC - this is for the new IARU HF Contest. This is an error in the IARU-Society information. The JARL contests report NR as exchange errors.